resources/metadata/helmExecute.yaml

```diff
@@ -264,6 +260,22 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: deploymentName
+        type: string
+        description: Defines the deployment name to use from helm package/publish commands.
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
```

Comment:
maybe rename parameter to `version` and pre-fill with value from `commonPipelineEnvironment` -> `artifactVersion`

We need to make sure that the automatic versioning executed in a pipeline is respected here.

Reply:
done